keyboards/4pplet/waffling60/rev_a/rev_a.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "quantum.h"
+
+#define RGB_OFF 0,0,0
+#define RGB_WHITE 255, 255, 255
+#define RGB_RED 255, 0, 0
+#define RGB_ORANGE 255, 20, 0
+#define RGB_YELLOW 255, 255, 0
+#define RGB_GREEN 0, 255, 0
+#define RGB_CYAN 0, 255, 255
+#define RGB_BLUE 0, 0, 255
+#define RGB_PURPLE 255, 0, 255
+#define RGB_PINK 255, 20, 20
+
+// set LOCK_INDICATORS to false if not used or if leds are used for custom function
+// do not use both, lock indicators will override layer indicator.
+#define LOCK_INDICATORS false // indicate capslock, scroll lock etc with rgb led
+#define LAYER_INDICATOR true // indicate layer with rgb led, functions may need modification.
+
+#define BRIGHTNESS_LEVELS 10
+#define DEFAULT_BRIGHTNESS_LEVEL 5
+
+#define TIMER_TOP 0xFFFFU
+
+bool led_update_kb(led_t led_state);
+bool led_update_user(led_t led_state);
+void keyboard_pre_init_user(void);
+void led_set_color(uint8_t r, uint8_t g, uint8_t b);
+void init_led(void);
+uint16_t cie_lightness(uint16_t v);
+
+// variables
+uint16_t brightness_level;
+
+enum waffling_layers {
+  _BASE1,
+  _BASE2,
+  _FN1,
+  _FN2,
+};
```

Comment:
This block should be removed from this file and added to both `keyboards/4pplet/waffling60/keymaps/default/keymap.c` and `keyboards/4pplet/waffling60/keymaps/via/keymap.c`, between `#include QMK_KEYBOARD_H` and `const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS]`.

```suggestion
```

Reply:
Thanks! 😄 I use the enum in waffling.c to detect which layer is active, to be able to indicate that with different color on the capslock RGB LED (indicate different function layers). If I move them to the keymap, these are no longer accessible. Do you have a suggestion on how to address that issue?